haystack/nodes/retriever/dense.py

```diff
@@ -1512,6 +1513,7 @@ def __init__(
                              - ``'sentence_transformers'`` (will use `_SentenceTransformersEmbeddingEncoder` as embedding encoder)
                              - ``'retribert'`` (will use `_RetribertEmbeddingEncoder` as embedding encoder)
         :param pooling_strategy: Strategy for combining the embeddings from the model (for farm / transformers models only).
+                             - ``'openai'``: (will use `_OpenAIEmbeddingEncoder` as embedding encoder)
```

Comment:
This is probably supposed to be part of `model_format` instead of `pooling_strategy`.